tests/test_collapse.py

```diff
@@ -76,13 +74,51 @@ def test_reconcile_prefix(self):
         """Test curie reconciliation is performing as expected."""
         msdf = parse_sssom_table(data_dir / "basic3.tsv")
 
-        with open(prefix_recon_yaml) as pref_rec:
-            prefix_reconciliation = yaml.safe_load(pref_rec)
+        self.assertEqual(
+            {
+                "a": "http://example.org/a/",
+                "b": "http://example.org/b/",
+                "c": "http://example.org/c/",
+                "d": "http://example.org/d/",
+                "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                "owl": "http://www.w3.org/2002/07/owl#",
+                "orcid": "https://orcid.org/",
+                "semapv": "https://w3id.org/semapv/vocab/",
+                "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                "skos": "http://www.w3.org/2004/02/skos/core#",
+                "sssom": "https://w3id.org/sssom/",
+            },
+            msdf.prefix_map,
+        )
+        prefix_reconciliation = {
+            "prefix_synonyms": {
+                "a": "c",
+                "b": "bravo",
+                "r": "rdfs",
+                "o": "owl",
+            },
+            "prefix_expansion_reconciliation": {
+                "c": "http://test.owl/c/",
+                "bravo": "http://test.owl/bravo",
+                "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                "owl": "http://www.w3.org/2002/07/owl#",
+            },
+        }
 
         recon_msdf = reconcile_prefix_and_data(msdf, prefix_reconciliation)
```

Comment:
I know this is not part of this PR, but consider turning the `prefix_reconciliation` object into a proper class instance for better documentation. Having the client need to construct a specifically formed yaml may not be the best way to ensure their happiness :P

Reply:
Agreed